mirror-plugins

```diff
@@ -88,7 +96,15 @@ projects.auto_paginate do |project|
       next project
     end
     puts "...creating and mirroring to it now"
-    plugin.create_github_repo
+    if dry_run?
```

Comment:
This only suppresses repo creation and pushing, so presumably the downloads still run in `dry-run` mode?

Reply:
Yes, because of the mechanism we'll be using to generate major version tags (in a forthcoming PR), we still need local copies of the repo even in dry run mode (as the dry run mode will in future, output what major version tags would be added, without actually adding them).